Mathlib/Analysis/Complex/ValueDistribution/FirstMainTheorem.lean

```diff
@@ -73,10 +154,15 @@ Second part of the First Main Theorem of Value Distribution Theory, qualitative
 meromorphic on the complex plane, then the characteristic functions for the value `⊤` of the
 function `f` and `f - a₀` agree asymptotically up to a bounded function.
 -/
-theorem abs_characteristic_sub_characteristic_shift_eqO (h : MeromorphicOn f ⊤) :
-    |characteristic f ⊤ - characteristic (f · - a₀) ⊤| =O[atTop] (1 : ℝ → ℝ) := by
-  simp_rw [isBigO_iff', eventually_atTop]
-  use log⁺ ‖a₀‖ + log 2, add_pos_of_nonneg_of_pos posLog_nonneg (log_pos one_lt_two), 0
-  simp [abs_characteristic_sub_characteristic_shift_le h]
+theorem isBigO_abs_characteristic_sub_characteristic_shift (h : MeromorphicOn f ⊤) :
```

Comment:
You have an `abs` in this theorem, but no `abs` in the theorem above. I think I would drop the `abs` in the lemma name, but also in the statements: for `IsBigO` theorems, it is equivalent to have them with or without abs, and it's slightly more readable without I think.